README.md

```diff
@@ -6,3 +6,20 @@ This is a place for style configurations for [Rainforest QA](https://www.rainfor
 
 - Make sure you're an owner on rubygems.org
 - `rake release`
+
+
+## Adding `rf-stylez` to a new project
+
+Create a `.rubocop.yml` in the root project directory and paste the following:
+```yml
+inherit_gem:
+  rf-stylez: ruby/rubocop.yml
+Style/HashSyntax:
+  Enabled: true
```

Comment:
You don't really need line 17-18 unless you want that particular cop enabled in your repo.